verification/cmd/verification-service/README.md

```diff
@@ -21,6 +22,33 @@ configuration:
 
 The verifier currently doesn't support any configuration.
 
+### Session manager configuration
+
+Session manager has a single configuration point: `backend`. This specifies
+which `ISessionManager` implementation will be used. The following backends are
+supported:
+
+- `ttlcache`: the default; this creates the session cache in memory of the
+  `verification-service` process.
+- `memcached`: uses an external [memcached](https://www.memcached.org/) server.
+
+All other entries under `sessionmanager` must be backend names (i.e. `ttlcache`
+or `memcached`), providing backend-specific configuration. Only configuration
+for the backend selected by `backend` entry will actually be used.
+
+#### `ttlcache` backend
```

Comment:
Interestingly this section does not specify where these changes needs to be done?

Below section in the README specifies/lists two config files, one for native (which is no longer used) and one for docker which is still relevant. Shall we not specify the config file for Verification using AWS where these setting are applicable?